imgee/templates/delete.html.jinja2

```diff
@@ -9,9 +9,9 @@
   {% endblock %}
   <div class="alert alert--error">
     <p class="mui--text-title">{{ faicon(icon='trash-alt', icon_size='subhead', baseline=true) }} Confirm delete</p><hr/>
-    <p class="mui--text-subhead">Are you sure you want to delete "<strong>{{file.title}}</strong>"?</p>
+    <p class="mui--text-subhead">{% trans file=file.title %}Are you sure you want to delete "<strong>{{ file }}</strong>{% endtrans %}"?</p>
     <p id="warning">
-      {{ faicon(icon='info-circle', baseline=true) }}&nbsp;<strong>Warning: If you delete it, you will lose all the links wherever this image appears.</strong>
+      {{ faicon(icon='info-circle', baseline=true) }}&nbsp;<strong>{% trans %}Warning: If you delete it, you will lose all the links wherever this image appears.{% endtrans %}</strong>
```

Comment:
Needs better text here: "Warning: pages that use this image will need to be updated."